_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -292,10 +294,35 @@ to start digging into our data proper. Check out the first few lines:
 head(gapminder)
 ```
 
+> ## Challenge 3
+>
+> It's good practice to also check the last few lines of your data and some in the middle. How would you do this?
+>
+> Searching for ones specifically in the middle isn't too hard but we could simply ask for a few lines at random. How would you code this?
+>
+> > ## Solution to Challenge 3
+> > To check the last few lines it's relatively simple as R already has a function for this:
+> > 
+> > ~~~
+> > tail(gapminder)
+> > tail(gapminder, n = 15)
+> > ~~~
+> > {: .r}
+> > 
+> > What about a few arbitrary rows just for sanity (or insanity depending on your view)?
+> > 
+> > ~~~
+> > gapminder[sample(nrow(gapminder), 5), ]
```

Comment:
Perhaps add a little more explanation to this syntax -- it might be a little much to parse for novices.

Reply:
Thanks, will do.